pages/common/pass.md

```diff
@@ -24,6 +28,6 @@
 
 `pass generate -c {{path/to/data}} {{num}}`
 
-- Run any git command against the underlying store repository:
+- Initialize a new Git repostitory (any changes done by pass will be comitted automatically):
```

Comment:
I think the original is a bit easier personally, however I'm fine with this. Just a typo in `repository`:
```suggestion
- Initialize a new Git repository (any changes done by pass will be comitted automatically):
```